starter-code/functions-and-arrays.js

```diff
@@ -34,7 +58,11 @@ console.log(total);
 
 // Calculate the Average
 function averageNumbers (array) {
-
+  var totalSum=0;
+  array.forEach(function(number){
```

Comment:
Do you have another function that sums all the values in an array? Then, maybe you can use it here instead of repeating code.